### PR TITLE
QDevil QDAC-II driver

### PR DESCRIPTION
**Pull request by jpsecher**

### CODE REVIEW

**Comment by FarBo** [comment]
Hi @jpsecher 
Thanks for the PR. Let's try to fix the issues in CI tests (issues seems in building docs). Let me know if you need help fixing them.

After that, we will try to review the PR.

**Comment by jpsecher** [comment]
I don't know how to fix the doc build, so I will remove all documentation for now.

**Comment by jenshnielsen** [comment]
@jpsecher That seems like a shame. You can disable the execution of the notebooks as documented here https://qcodes.github.io/Qcodes/examples/writing_drivers/Creating-Instrument-Drivers.html#Documentation  I think that should be enough to get it building

**Comment by FarBo** [comment]
@jpsecher 

I started looking into the QDAC2.py. As the first general comment, It would be nice if you could add docstring min. for public functions/ classes.
Also, since some of the arguments are static-typed and some not, Is it possible to type those type-missing arguments?

**Comment by jpsecher** [comment]
Hi, I believe I have made all the improvements that I can, thanks!

File: qcodes_contrib_drivers/drivers/QDevil/QDAC2.py [hunk 1]
```diff
@@ -0,0 +1,2171 @@
...
+    def _channel_message(self, template: str) -> None:
+        return self._channel._channel_message(template)
+
+
+class _Dc_Context(_Channel_Context):
+
+    def __init__(self, channel: 'QDac2Channel'):
+        super().__init__(channel)
+        self._write_channel('sour{0}:dc:trig:sour hold')
+        # TODO: should be sum type of Internal/External?  rename to start_on
```
Comment (astafan8) [inline qcodes_contrib_drivers/drivers/QDevil/QDAC2.py:196]: is the todo still valid? and the "rename ..." comment?

File: qcodes_contrib_drivers/drivers/QDevil/QDAC2.py [hunk 2]
```diff
@@ -0,0 +1,2171 @@
...
+
+
+class Measurement_Context(_Channel_Context):
+
+    def __init__(self, channel: 'QDac2Channel', delay_s: float,
+                 repetitions: int, current_range: str,
+                 aperture_s: Optional[float], nplc: Optional[int]):
+        super().__init__(channel)
+        self._trigger_start: Optional[QDac2Trigger_Context] = None
+        #self._write_channel('sens{0}:trig:sour hold')
```
Comment (astafan8) [inline qcodes_contrib_drivers/drivers/QDevil/QDAC2.py:941]: should this be removed or uncommented?

**Review by astafan8 — APPROVED** [review]
@jpsecher sorry for the delay in review! I had a look and it's ready to merge. Just left a few clarification comments on the todo items in the code.

File: qcodes_contrib_drivers/drivers/QDevil/QDAC2.py (hunk 2 above)
Comment (astafan8) [reply qcodes_contrib_drivers/drivers/QDevil/QDAC2.py]: leaving it for now

**Pull request merged**